data/filetypes.Schema.conf

```diff
@@ -0,0 +1,66 @@
+# For complete documentation of this file, please see Geany's main documentation
+[styling]
```

Comment:
You should inherit Python's styles as you use the Python lexer, using `[styling=Python]`